specification/dimensions/adjustmenttype.md

```diff
@@ -0,0 +1,54 @@
+# adjustment type
+
+Adjustment Type indicates the record represents a change to your monthly invoice through after-the-fact adustements (e.g. refunds, pricing changes). An adjustment can be positive or negative.
+Adjustment Type can be used by practitioners to assess any impacts to spend from positive or negative adjustments.
+
+The AdjustmentType column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values.
+
+- Usage Correction
+- Price Correction
+- GoodWill
+- SLA Violation
+- Balance Transfer
+- General Adjustment
+- Promo Credit
+- Refund - Reservations
+- Refund - Savings Plan
+- Refund - General usage
```

Comment:
We should only have one of these. We have other columns to indicate whether it's related to a commitment. Also "reservation" and "savings plan" are provider-specific terms.

Reply:
This was to show a refund could be multiple types of refund. In Azure, reservation refunds show up in reservations transactions and not usage details. So thought we should maybe call out different refund types.

Reply:
Understood. My point is that when a refund is for a reservation, the provider should indicate that in the CommitmentDiscountId/Name/Type/Plan columns.

Reply:
We have an underlying goal to make single-purpose columns that are easy to use for summarization and filtering. In this case, we'd use AdjustmentType to identify refunds and then other columns to identify what the refund was for.